python/dolfinx/common.py

```diff
@@ -91,7 +102,17 @@ def elapsed(self):
 
 
 def timed(task: str):
-    """Decorator for timing functions."""
+    """Decorator for timing functions. The basic usage is::
+
+        @timed('Description of function')
```

Comment:
Google doctring style uses `Examples:` section for code example usage, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html.